gcapi/apibase.py

```diff
@@ -176,3 +182,15 @@ def partial_update(self, pk, **kwargs):
 
     def delete(self, pk):
         return (yield from self.perform_request("DELETE", pk=pk))
+
+
+class PageResult(list):
```

Comment:
Just out of curiosity. Did you find any references on subclassing from list vs UserList?

Reply:
It was a bit of a mixed bag. Python2 had a strict suggestion to use UserList, this is not the case for python 3. 

Depending on what behaviour you are looking for the common denotion is now to use a collections (e.g. UserList, or Sequence). I felt it should be a minimum change of the interface so I went with `list`. This did mean that the `count()` couldn't be overwritten, hence the `total_count`.